client/README.md

```diff
@@ -1,11 +1,25 @@
-# React + TypeScript + Vite
+# Aqua Academy Client
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+## Test starting a quiz
+Go to `/quiz/login/sdg-14-quiz` in order to test login page. For now, we're using mock data to simulate starting a quiz.
```

Comment:
Vanwaar deze logica?

is het niet een component waarbij je alleen de slug (genereerde code) hoeft in te vullen en dan naar /quiz/`HIER-DAN-ID-OF-IETS`?

Reply:
Het idee was om te voorkomen dat willekeurige mensen een quiz kunnen beginnen, net zoals kahoot. In mijn scenario geeft een docent een url en een wachtwoord/inlogcode waarmee de leerlingen een quiz kunnen starten. De url is er zodat ze bij de juiste quiz komen. En het wachtwoord/inlogcode om te voorkomen dat ongeauthoriseerden mensen een quiz kunnen starten. 

In mijn code is de inlogcode gelijk aan de _id. Wat onhandig is, omdat mongodb een moeilijk te onthouden code genereerd. Dit ga ik nog wijzigen naar een extra property genaamd `password`.